.npmrc

```diff
@@ -1,2 +0,0 @@
-package-lock=false
-shrinkwrap=false
```

Comment:
Question: Are we allowing shrinkwrap too?

Reply:
Good point, apparently not! I'll revert these cases where I've deleted the option.